secure_message/repository/database.py

```diff
@@ -169,3 +169,20 @@ def __init__(self, msg_id='', event=''):
         self.msg_id = msg_id
         self.event = event
         self.date_time = datetime.now(timezone.utc)
+
+
+class Conversation(db.Model):
+    """Conversation table model"""
+    __tablename__ = "conversation"
+
+    id = Column('id', String(length=60), primary_key=True)
```

Comment:
Random Q how come in every other table we set the id to a Integer but here its set to a String?

Reply:
I'll mention it here incase anyone else asks.  This table will, at some point, represent the conversation.  Theres no point in calling the id of it 'conversation_id' because we're in the conversation table already so we know what the ID is for already.  So we'll just call it id

Reply:
if the id is going to be used in the queries liek we do for the thread id of msg id we should create an index.

Reply:
Index added